core/src/main/kotlin/configuration.kt

```diff
@@ -100,6 +100,7 @@ interface DokkaConfiguration : Serializable {
     val pluginsConfiguration: List<PluginConfiguration>
     val delayTemplateSubstitution: Boolean
     val suppressObviousFunctions: Boolean
+    val includes: Set<File>
```

Comment:
I'm not sure if this is the best place for this property. This configuration is AFAIK shared with single- and multi-module tasks and single-module tasks already have a configuration option with this name in the source sets. We can still add it here though, but we should support it in the single-module then